src/components/InputDropdown/index.js

```diff
@@ -34,6 +34,11 @@ function InputDropdown({
   );
   return (
     <div css={styles.root}>
+      {placeholder && (
```

Comment:
Apologies for not catching this the first time through, but can we change the markup and prop names in here to match the other inputs? Replacing the outer div with a `label`, changing the `placeholder` prop to `label`, etc? Keeping these consistent would be 👍